match/ouachita_da.py

```diff
@@ -0,0 +1,42 @@
+from datamatch import ThresholdMatcher, JaroWinklerSimilarity, ColumnsIndex
+from lib.path import data_file_path
+import pandas as pd
+import sys
+sys.path.append('../')
+
+
+def prepare_lsp_and_opso_post_data():
+    post = pd.read_csv(data_file_path('clean/pprr_post_2020_11_06.csv'))
+    return post[post.agency.isin(['la state police', 'ouachita parish so'])]
+
+
+def match_cprr_with_lsp_and_opso_post(cprr, post):
+    dfa = cprr[['uid', 'first_name', 'last_name', 'agency']]
+    dfa.loc[:, 'fc'] = dfa.first_name.fillna('').map(lambda x: x[:1])
+    dfa = dfa.drop_duplicates(subset=['uid']).set_index('uid')
+
+    dfb = post[['uid', 'first_name', 'last_name', 'agency']]
+    dfb.loc[:, 'fc'] = dfb.first_name.fillna('').map(lambda x: x[:1])
+    dfb = dfb.drop_duplicates(subset=['uid']).set_index('uid')
+
+    matcher = ThresholdMatcher(ColumnsIndex(['fc']), {
+        'last_name': JaroWinklerSimilarity(),
+        'first_name': JaroWinklerSimilarity(),
+        'agency': JaroWinklerSimilarity(),
```

Comment:
Are there lots of misspelling/typos for `agency`?

Reply:
They're all missing

Reply:
Sorry I'm missing the answer, are there typos for `agency`?

Just a reminder, these fields are all fuzzy-matched. While fuzzy matching is great for matching values with possible typos, it is a lot slower than simple `==` comparison. I find it quite odd that the `agency` field is also fuzzy-matched. Even if the matching is perfect, are you left with agency values with typos in them? And if there are no typos, you might want to put `agency` in `ColumnsIndex` instead which not only not making it slower, it actually makes matching faster.